libs/feature/map/src/lib/map-context/map-context.service.ts

```diff
@@ -112,13 +116,22 @@ export class MapContextService {
           }),
         })
       case MapContextLayerTypeEnum.WMS:
-        return new TileLayer({
-          source: new TileWMS({
-            url: layerModel.url,
-            params: { LAYERS: layerModel.name },
-            gutter: 20,
-          }),
-        })
+        if (mapConfig?.DO_NOT_TILE_WMS) {
+          return new ImageLayer({
+            source: new ImageWMS({
+              url: layerModel.url,
+              params: { LAYERS: layerModel.name, TILED: false },
```

Comment:
```suggestion
              params: { LAYERS: layerModel.name },
```
This param can be omitted safely, we usually don't include it if it's false